src/popup/components/Modals/AccountImport.vue

```diff
@@ -102,10 +116,23 @@ export default defineComponent({
       router.push(store.state.loginTargetLocation);
     }
 
+    async function openScanQrModal() {
+      const scanResult = await openModal(MODAL_READ_QR_CODE, {
+        title: t('pages.index.scanSeedPhrase'),
+        icon: 'critical',
+      });
+
+      if (!scanResult) {
+        return;
+      }
+      mnemonic.value = scanResult;
```

Comment:
```suggestion
      if (scanResult) {
        mnemonic.value = scanResult;
      }
```
Early return in this case makes no sense.